go/bundle/bundle.go

```diff
@@ -138,6 +139,12 @@ type requestEntryWithOffset struct {
 	Offset uint64
 }
 
+type section interface {
+	Name() string
+	Len() int
+	Bytes() []byte
+}
```

Comment:
Can we use `WriteTo` instead of getter property interface?

https://golang.org/pkg/io/#WriterTo

Reply:
Done.

Reply:
Ah sorry but I meant to remove section interface and use `io.WriterTo` instead

Reply:
Hmm. We need to write (name, length) pair and section content separately, so just a `WriterTo` doesn't suffice.

Reply:
Oh I see.

In this case, I prefer to define e.g. `MarshalCbor` function that takes cbor's encoder to having getters, but that's optional.

Reply:
Let me merge this as-is, and do the refactoring separately.